stable/kommander/Chart.yaml

```diff
@@ -1,10 +1,10 @@
 apiVersion: v1
-appVersion: 1.3.0
+appVersion: 1.2.0
```

Comment:
1.3.0 to 1.2.0?

Reply:
yeah this corresponds to konvoy version. cert-manager upgrade is going into 1.2 so bumping this back down to 1.2. after this PR we'll bump the chart version up to 0.13.x and appVersion back up to 1.3 for next konvoy release